src/main/java/org/shanerx/tradeshop/Potions.java

```diff
@@ -0,0 +1,299 @@
+/*
+ *     Copyright (c) 2016-2017 SparklingComet @ http://shanerx.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *              http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * NOTICE: All modifications made by others to the source code belong
+ * to the respective contributor. No contributor should be held liable for
+ * any damages of any kind, whether be material or moral, which were
+ * caused by their contribution(s) to the project. See the full License for more information
+ */
+
+package org.shanerx.tradeshop;
+
+import org.bukkit.Material;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.PotionMeta;
+import org.bukkit.potion.PotionData;
+import org.bukkit.potion.PotionType;
+
+@SuppressWarnings("unused")
+public enum Potions {
```

Comment:
Maybe we want to call this PotionType or something similar, to comply to enum naming conventions (singular nouns).

Reply:
PotionType is already used in bukkit to i wanted to avoid that, and so is Potion. Potions is used in NMS but we will never need to touch that so having this doesnt mean we have 2 enums with the same name

Reply:
You can call it TSPotion or something like that.
Not that important though.

Reply:
Im going to leave it for now just because i dont want to go through everything and update the name, maybe in a cleanup later, but when we're doing 2.0 it will definitely get changed